tests/rustdoc-ui/unescaped_backticks.stderr

```diff
@@ -280,11 +280,11 @@ LL | |     /// level changes.
    | |______________________^
    |
    = help: the opening backtick of a previous inline code may be missing
-            change: The Subscriber` may be accessed by calling [`WeakDispatch::upgrade`],
-           to this: The `Subscriber` may be accessed by calling [`WeakDispatch::upgrade`],
+            change:  The Subscriber` may be accessed by calling [`WeakDispatch::upgrade`],
```

Comment:
This change seems surprising, no?

Reply:
It's strange indeed, especially because (based on the logging I'm looking at), the indent is correctly assessed to be 1. Looking into this more...

Reply:
Ok, so here's the reason. It's because this code is wrapped in a macro (which is just the identity function, returning exactly its input). However, this turns the sugared doc attrs into raw doc attrs; thus, the indent is computed as 0. This also makes the lint's suggestion go into the "uglier" path because `source_span_for_markdown_range` requires all inputs to be sugared doc attrs.

I don't know if there's something we want to change here. It does seem unfortunate that the indentation rules will change if the docs are passed through a macro, although that does seem like a rare occurrence.

Reply:
As you prefer then! r=me if it sounds good to you as is. ;)